includes/free-trial/class-wc-calypso-bridge-free-trial-orders-notice.php

```diff
@@ -4,7 +4,7 @@
  * Class WC_Calypso_Bridge_Free_Trial_Orders_Notice.
  *
  * @since   2.0.5
- * @version 2.0.8
+ * @version 2.0.15
```

Comment:
```suggestion
 * @version 2.0.16
```

Looks like 2.0.15 has just been released.

https://github.com/Automattic/wc-calypso-bridge/commit/a15c184837a30beb213e4b312dc1ed226c0b43ab